### ISSUE TITLE
Unable to export EC2 image

### ISSUE THREAD

**Issue by geopcgeo** [BauerLab/ngsane#51]
I am trying to up this Image on my local by referring this [link](https://github.com/BauerLab/ngsane/wiki/How-to-use-the-virtual-machine). Please let me know how to do this ec2-create-instance-export-task command on "[Export the EC2 image as Virtual Machine image see instructions](http://aws.amazon.com/ec2/vm-import/)" 

We up a instance with AMI image ami-75721c4f and then on another machine installed ec2 tool and executed this step after giving proper permission to bucket:

`ec2-create-instance-export-task i-9cb45524 -e VMware -f VMDK -c OVA  -b <bucketname> -O <key> -W <secretkey> --region ap-southeast-2`

But getting this error:

`Client.NotExportable: Only imported instances can be exported.`

So can you please help on this.

**Comment by BauerLab** [comment]
Yes that's correct, the requirements have changed. I will look into creating an updated AMI.

**Comment by geopcgeo** [comment]
Thanks for the update. In the mean time we tried installing setting up of Ngsane on an Ubuntu Machine by referring this [Url](https://github.com/BauerLab/ngsane/wiki/How-to-setup-ngsane)

We executed the [shell script](https://github.com/BauerLab/ngsane/blob/master/doc/installation/setup) for bioinformatics software but on last line we are getting an error. In which the url to download referenceData.tgz is not working (wget http://www.hpsc.csiro.au/users/bau04c/datahome/Sandbox/smokebox_ngsane/smokebox/referenceData.tgz)

So can you please let me know the new location and help us on this.

Thanks
